### ISSUE TITLE
Can we get the relative distance of all cells?

### ISSUE THREAD

**Issue by YiweiNiu** [rajewsky-lab/novosparc#30]
Hi,

Thank you for this great package!

I wonder if we could get the relative distance of all cells. If I understand correctly, `novoSpaRc` returns the expression data on spatial locations, and it may calculate the "physical distance" between cells.

How can we get this information?

Sorry if my question is too naive or due to my ignorance.

Many thanks!
Yiwei Niu

**Comment by nukappa** [comment]
Hi, can you please explain in more detail what you mean by "relative distance of all cells"? `novoSpaRc` will map the cells spatially and provide the spatial DGE matrix. The "physical distances" between the mapped locations are given, as you provide the target space for the reconstruction.

**Comment by YiweiNiu** [comment]
Thank you for your reply!

Sorry I did not make myself clear. I mean, for example, for cell A and cell B, can we get the spatial distance between them?

By viewing the final spatial expression of specific marker genes, I could see cells of group A and cells of group B are close. But it is a visual check, what I want to do is to qualify the distances  between groups of cells, i.e. to compare if the distance between group A and group B is smaller than that of group A and group C.

Hope that I did not misunderstand the purpose of `novosparc`.
